src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseTestService.java

```diff
@@ -647,8 +647,7 @@ public void startProgrammingExercise_correctInitializationState(ExerciseMode exe
             participant = setupTeam(user);
         }
         mockDelegate.mockConnectorRequestsForStartParticipation(exercise, participant.getParticipantIdentifier(), participant.getParticipants(), true, HttpStatus.CREATED);
-        final var path = ParticipationResource.Endpoints.ROOT + ParticipationResource.Endpoints.START_PARTICIPATION.replace("{courseId}", String.valueOf(course.getId()))
-                .replace("{exerciseId}", String.valueOf(exercise.getId()));
+        final var path = ParticipationResource.Endpoints.ROOT + '/' + ParticipationResource.Endpoints.START_PARTICIPATION.replace("{exerciseId}", String.valueOf(exercise.getId()));
```

Comment:
What is the difference between `ParticipationResource.Endpoints.ROOT` and just `ROOT`? Can we use `ROOT` for consistency?